chromium_src/BUILD.gn

```diff
@@ -45,6 +45,25 @@ config("chromium_src_config") {
   }
 }
 
+if (is_win) {
+  # This is in a separate config so the flags can be applied to dependents.
+  # ldflags in GN aren't automatically inherited.
+  config("browser_win_linker_flags") {
```

Comment:
this doesn't need to be copied, you can just use `//chrome/browser:browser_win_linker_flags` below